java-frontend/src/test/files/se/NullDereferenceCheck.java

```diff
@@ -714,3 +714,11 @@ private static void bar(Object o, byte[] b) {
     FooBar.bar(null, new byte[10]); // Compliant
   }
 }
+
+class Params {
+
+  void foo(int a) {
```

Comment:
Not sure how this testcase actually relates to the ticket. And I am not even sure how this case actually demonstrate anything related to primitive : you can write the same test with Object instead of int and it will pass...